packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routing.js

```diff
@@ -51,7 +51,7 @@ export default /* @ngInject */ ($stateProvider) => {
         }),
 
       onAssociateIpBlock: /* @ngInject */ ($state) => () =>
-        $state.go('app.dedicatedCloud.details.dashboard.associate-ip-bloc'),
+        $state.href('app.dedicatedCloud.details.dashboard.associate-ip-bloc'),
```

Comment:
related to this :
https://github.com/ovh/manager/pull/6890/files#r848529281

Reply:
I renamed this function.